msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java

```diff
@@ -4159,7 +4161,7 @@ public String getAuthorString()
     
     public String getPlacementId() 
     {
-       return Validator.escapeJavascript("Main" + toolManager.getCurrentPlacement().getId());
+       return formattedText.escapeJavascript("Main" + toolManager.getCurrentPlacement().getId());
```

Comment:
I'm not sure this is right. Validator.escapeJavascript is marked as deprecated, but the doc says to use StringEscapeUtils (Apache) instead. FormattedText doesn't use StringEscapeUtils. I'd either leave Validator here, or move FormattedText to use Apache (with the testing that'll need).

Reply:
copmmons-text: StringEscapeUtils.escapeEcmaScript(String)?